applications/crossbar/src/modules_v2/cb_phone_numbers_v2.erl

```diff
@@ -460,6 +547,13 @@ normalize_view_results(JObj, Acc) ->
      | Acc
     ].
 
+%% @private
+-spec normalize_port_view_result(kz_json:object()) -> kz_json:object().
+normalize_port_view_result(JObj) ->
+    Number = kz_json:get_value([<<"key">>, 2], JObj),
```

Comment:
@icehess we should try to document these things or make use of macros that show what it means.